### PR TITLE
docs: add hint that the generated data might be real ones

### PR DESCRIPTION
**Pull request by ST-DDT**
We should warn our users that the generated data may be working ones and not the well known `555` dummy ones.

### CODE REVIEW

**Comment by Shinigami92** [comment]
Random thought: why not for:
- email
- iban
- vrm
- vim
- bitcoin address
- ipv4/6
- ...

I'm not sure if this hint is helpful :thinking:

**Comment by ST-DDT** [comment]
We might want to add a general hint at some point (or maybe actually generate fake numbers).

But I created this one to address the issue raised in Discord: 
https://discord.com/channels/929487054990110771/929544565348777984/974285076064002148

**Comment by Shinigami92** [comment]
> We might want to add a general hint at some point (or maybe actually generate fake numbers).
> 
> But I created this one to address an issue raised in Discord: [discord.com/channels/929487054990110771/929544565348777984/974285076064002148](https://discord.com/channels/929487054990110771/929544565348777984/974285076064002148)

IMO we should just add the hint now globally. For me it doesn't make really sense to add it here specifically just because one person asked in discord if values are real or not.
Some day just the next person would come up and ask for another function and so on :shrug: 
I also would say it is not in scope for Faker at all to generate or prevent real data. Just data that "looks" real and if they are "accidentally" real, so they are.

**Review by prisis — DISMISSED** [review]

**Comment by pkuczynski** [comment]
I agree with @Shinigami92 that it should be done as a general hint and not for a specific function. The whole concept of faker is to generate data that looks real, so as he pointed out, they might be actually real!

**Comment by ST-DDT** [comment]
Yeah, I will think of a way to put it somewhere else.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,10 @@
 - 🧍 Names - Generate virtual humans with a complete online and offline identity.
 - 🔢 Numbers - Of course, we can also generate random numbers and strings.
 
+> **Note**: Faker tries to generate life like data and not specifically test data.
+> So the generated names, addresses, emails, phone numbers or other data might be existing and coincidentally valid information.
```
Comment (ST-DDT) [inline README.md:34]: Not sure about this sentence.
I hope the intention is clear, but the wording is probably not the best.

Comment (import-brain) [reply README.md:34]: What about "Faker tries to generate realistic data and not obvious fake data. The generated names, addresses, emails, phone numbers, and/or other data might be coincidentally valid information."?

**Review by Shinigami92 — APPROVED** [review]

**Review by ejcheng — APPROVED** [review]

**Pull request merged**